### ISSUE TITLE
Path appears invalid  in  tt_at_path(tt, rowpath) 

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/rtables#596]
```R
bug_example <-function(adsl, adrs, arm = "TRT01P", refgroup = NULL, method = "wilson",
                        level = c("CR", "PR", "SD", "PD"),
                        label = c("CR (%)", "PR (%)", "SD (%)", "PD (%)"),
                        show_level = TRUE){


  slref_arm <- sort(unique(adsl[[arm]]))
  anl_arm <- sort(unique(adrs[[arm]]))


  anl <- adrs %>%
    select(STUDYID, USUBJID, AVALC) %>% # unique %>%
    dplyr::mutate(
      is_rsp = AVALC %in% c("CR", "PR"),
      AVALC = as.character(AVALC),
      AVALC = ifelse(AVALC %in% level, AVALC, "<Missing>"),
      AVALC = factor(AVALC, levels = c(level, "<Missing>"),
                     labels = c(label, "<Missing>")),
      AVALC1 = ifelse(is_rsp, "Responders", "<Missing>"),
      AVALC1 = factor(AVALC1, levels = c("Responders", "<Missing>"),
                      labels = c("Responders", "<Missing>")
      )
    ) %>%
    inner_join(., adsl, by = c("STUDYID", "USUBJID")) %>%
    df_explicit_na(char_as_factor = FALSE)

  lyt_01 <- basic_table() %>%
    split_cols_by(
      var = arm,
      ref_group = refgroup
    ) %>%
    add_colcounts() %>%
    analyze(
      vars = "is_rsp",
      afun = s_proportion_1,
      extra_args = list(method = method),
      table_names = "est_prop",
      format = list(n_prop = "xx (xx.x%)", prop_ci = "(xx.x, xx.x)"),
      show_labels = "hidden"
    )
  if (show_level){
    lyt_01 <- lyt_01 %>%
      summarize_vars(
        vars = "AVALC",
        .stats = "count_fraction",
        .formats = NULL,
        na_level = "<Missing>",
        denom = "N_col",
        show_labels = "hidden"
      )
  }

  if (!is.null(refgroup)) {
    lyt_01 <- lyt_01 %>%
      estimate_proportion_diff(
        vars = "is_rsp",
        show_labels = "hidden",
        var_labels = "Unstratified Analysis",
        table_names = "est_prop_diff",
        .stats = c("diff"),
        .labels = c(diff = "ORR difference"),
        .formats = c(diff = "xx.xx")
      )
  }

  result1 <- build_table(lyt = lyt_01, df = anl, alt_counts_df = adsl)

  if (!is.null(refgroup)){
    result <- rbind(result1[1,],
                    result1[2:(nrow(result1)-1), ] %>% indent(., 1),
                    result1[nrow(result1), ])
  } else {
    result <- rbind(result1[1,],
                    result1[2:nrow(result1), ] %>% indent(., 1))
  }
  result
}


library(scda)
library(tern)
library(dplyr)
adsl <- synthetic_cdisc_dataset("latest", "adsl") %>%
  mutate(TRT01P = factor(TRT01P, levels = c("A: Drug X", "B: Placebo", "C: Combination")))
adrs <- synthetic_cdisc_dataset("latest", "adrs") %>%
  mutate(TRT01P = factor(TRT01P, levels = c("A: Drug X", "B: Placebo", "C: Combination"))) %>%
  filter(PARAMCD=='INVET')
tbl <- bug_example(adsl, adrs, "TRT01P")
row_paths(tbl)
cell_values(tbl, rowpath = c("rbind_root", "ma_est_prop_AVALC", "est_prop", "n_prop"))
cell_values(tbl, rowpath = c("rbind_root", "ma_est_prop_AVALC", "est_prop", "prop_ci"))

Error in tt_at_path(tt, rowpath) : 
  Path appears invalid for this tree at step prop_ci
```

**Comment by gmbecker** [comment]
This would be fixed by #864 @Melkiades @shajoezhu

**Comment by Melkiades** [comment]
tracked by #864
